src/code/debug_malloc.c

```diff
@@ -97,8 +97,8 @@ void* DebugArena_Calloc(u32 num, u32 size) {
 
 #if OOT_DEBUG
 void DebugArena_Display(void) {
-    // "Zelda heap display" ("Zelda" should probably have been changed to "Debug")
-    PRINTF("ゼルダヒープ表示\n");
+    // ("Zelda" should probably have been changed to "Debug")
+    PRINTF(T("ゼルダヒープ表示\n", "Zelda heap display\n"));
```

Comment:
we can remove the parens now that its the only thing on the line. Maybe a new comment would be better altogether:
"This is likely a copy/paste from `ZeldaArena_Display`. It should say "Debug Heap".

Reply:
also I think Heap should be capitalized in all names like Zelda Heap, Debug Heap, Gamestate Heap. But that can be a different cleanup

Reply:
I can't locate "debug heap" or "game heap" so yeah let's do that another time

Reply:
guess i was thinking of system heap `"System heap display"`, doesnt seem like gamestate alloc has similar comments. but anyway sure